package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuxt-content-algolia",
-  "version": "0.2.0",
+  "version": "0.3.1",
```

Comment:
Repo seems to have a 0.3.0 release tag, but because the package.json is not updated NPM thinks it is still 0.2.0.

Suggest bumping to 0.3.1 to be clearer.